05-swiss-officers-more-introverted.ipynb

```diff
@@ -0,0 +1,684 @@
+{
```

Comment:
<p>QAP in dyadic_hypothesis_test is implemented wrong. See comment there. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gabrieltorresgamez/fhnw-san-project/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>